static/app/views/issueDetails/groupReplays/groupReplays.tsx

```diff
@@ -278,7 +278,7 @@ function GroupReplaysTable({
   );
 }
 
-const StyledLayoutPage = styled(Layout.Page)`
```

Comment:
just copy paste this tbh not worth exporting

Reply:
I think I need to export so I can overwrite the styles in the wrapper styled component `ReplayBody`